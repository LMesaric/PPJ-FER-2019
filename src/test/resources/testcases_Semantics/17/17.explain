semantic: integer constant to large
note: '2147483648' on line 112
