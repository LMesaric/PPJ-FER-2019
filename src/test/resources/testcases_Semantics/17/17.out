<primarni_izraz> ::= BROJ(112,2147483648)
